content/implementations/jvm.md

```diff
@@ -387,9 +395,9 @@ Definition.
 
 ```xml
 <dependency>
-    <groupId>info.cukes</groupId>
+    <groupId>io.cucumber</groupId>
     <artifactId>cucumber-groovy</artifactId>
-    <version>{{ site.versions.cucumber_jvm }}</version>
+    <version>${version}</version>
```

Comment:
May actually need two versions. One for the versions that are no longer part of core and one for the versions that are.

Reply:
Or we should (re)move the dependencies for the jvm languages that were removed from cucumber-jvm?